### ISSUE TITLE
Add separator option for "Parse CSV" workflow task

### ISSUE THREAD

**Issue by mana99** [aj3423/SpamBlocker#193]
### The feature

I am trying to use the [tellows API](https://www.tellows.co.uk/openapi/index.html) to populate the spam database with an automation workflow and I need to parse a CSV file with a semicolon separator, but as far as I know only the comma separator is accepted. It would be nice to be able to specify a different separator.
Thanks in advance

**Comment by aj3423** [comment]
Done in the action build: https://github.com/aj3423/SpamBlocker/actions/runs/11868582892
Now it detects separators automatically,  it supports:   `,`,  ` ;` and `|`

The tellows looks interesting, that csv file costs 19,95 €, you paid for it?

**Comment by mana99** [comment]
@aj3423 Thanks!

I haven't bought it yet, as I was going to test it first with the test API key in the documentation. I used to pay for the premium features on their app as I found that the database was quite comprehensive due to its crowd-sourced nature. The premium features cost roughly around 10€/year which is the same as the 2 years of API usage (although the premium might have the advantage to work in every country)
